<h2>Viral metagenomics and network analysis</h2>
<p>I propose the use of network analysis and concepts of island biogeography for the analysis of complex viromes. The structure of bipartite networks fits naturally with the association of virus and plant hosts, the ecological patterns of interaction are described as nested or modular given the structure of the virus community. A long-term goal of this work is to create a more quantitative approach for understanding community assembly rules and for predicting future emerging pathogens.</p>

<p>Virome dynamics respond to the distribution of plants, changes in the landscape for agricultural usage and climate change alter the patterns of distribution of hosts, and vectors, therefore the landscape connectivity allow emerging pathogens to establish in new niches. We are comparing host and location combinations to determine ecological patterns, viral positive and negative interactions, and spatial relationships.</p>

<h2>Diversity of viruses: Viral discovery</h2>

<p>Viral metagenomics is a useful tools for assessing the diversity of viruses. Using high-throughput sequencing (HTS) has become the standard tool for virus diagnosis, and the go tool to reveal viruses associated with plants. Viruses can interact in many ways with plants causing necrosis, mosaics, stunting symptoms and sometimes, some are asymptomatic. Another interesting feature is that this technique reveals multiple interactions that otherwise would have been difficult to identify, in most cases when an unknown virus is present within the same plant.</p>

<p>I am developing a framework for the classification of viruses starting from HTS data.</p>
