<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

<!-- Begin Jekyll SEO tag v2.5.0 -->
<title>Research | Ricardo I. Alcala Briseno - Virus disease ecology</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Viral metagenomics and network analysis I propose the use of network analysis and concepts of island biogeography for the analysis of complex viromes. The structure of bipartite networks fits naturally with the association of virus and plant hosts, the ecological patterns of interaction are described as nested or modular given the structure of the virus community. A long-term goal of this work is to create a more quantitative approach for understanding community assembly rules and for predicting future emerging pathogens." />
<meta property="og:description" content="Viral metagenomics and network analysis I propose the use of network analysis and concepts of island biogeography for the analysis of complex viromes. The structure of bipartite networks fits naturally with the association of virus and plant hosts, the ecological patterns of interaction are described as nested or modular given the structure of the virus community. A long-term goal of this work is to create a more quantitative approach for understanding community assembly rules and for predicting future emerging pathogens." />
<link rel="canonical" href="http://0.0.0.0:4000/research" />
<meta property="og:url" content="http://0.0.0.0:4000/research" />
<meta property="og:site_name" content="Ricardo I. Alcala Briseno - Virus disease ecology" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2000-01-02T00:00:00-05:00" />
<script type="application/ld+json">
{"description":"Viral metagenomics and network analysis I propose the use of network analysis and concepts of island biogeography for the analysis of complex viromes. The structure of bipartite networks fits naturally with the association of virus and plant hosts, the ecological patterns of interaction are described as nested or modular given the structure of the virus community. A long-term goal of this work is to create a more quantitative approach for understanding community assembly rules and for predicting future emerging pathogens.","@type":"BlogPosting","url":"http://0.0.0.0:4000/research","headline":"Research","dateModified":"2000-01-02T00:00:00-05:00","datePublished":"2000-01-02T00:00:00-05:00","mainEntityOfPage":{"@type":"WebPage","@id":"http://0.0.0.0:4000/research"},"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->

    <link rel="stylesheet" href="/assets/css/style.css?v=980dfdf57474e7298d834d0842036339988b3536">
  </head>
  <body>
    <div class="container-lg px-3 my-5 markdown-body">
      
      <h1><a href="http://0.0.0.0:4000/">Ricardo I. Alcala Briseno - Virus disease ecology</a></h1>
      

      <h2>Viral metagenomics and network analysis</h2>
<p>I propose the use of network analysis and concepts of island biogeography for the analysis of complex viromes. The structure of bipartite networks fits naturally with the association of virus and plant hosts, the ecological patterns of interaction are described as nested or modular given the structure of the virus community. A long-term goal of this work is to create a more quantitative approach for understanding community assembly rules and for predicting future emerging pathogens.</p>

<p>Virome dynamics respond to the distribution of plants, changes in the landscape for agricultural usage and climate change alter the patterns of distribution of hosts, and vectors, therefore the landscape connectivity allow emerging pathogens to establish in new niches. We are comparing host and location combinations to determine ecological patterns, viral positive and negative interactions, and spatial relationships.</p>

<h2>Diversity of viruses: Viral discovery</h2>

<p>Viral metagenomics is a useful tool for assessing the diversity of viruses. Using high-throughput sequencing (HTS) has become the standard tool for virus diagnosis, and the to go tool for revealing associations of viruses within the plant host. Viruses interactions occur at different levels within plant-host,  causing necrosis, mosaics, stunting symptoms and some times, asymptomatic. Another interesting feature is that this technique reveals multiple interactions that otherwise would have been difficult to identify, in most cases when an unknown virus is present within the same plant.</p>


      
      <div class="footer border-top border-gray-light mt-5 pt-3 text-right text-gray">
        This site is open source. <a href="https://github.com/ricardoi/ralcala.github.io/edit/gh-pages/_posts/2000-01-02-research.md">Improve this page</a>.
      </div>
      
    </div>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/4.1.0/anchor.min.js" integrity="sha256-lZaRhKri35AyJSypXXs4o6OPFTbTmUoltBbDCbdzegg=" crossorigin="anonymous"></script>
    <script>anchors.add();</script>
    
  </body>
</html>
